docs/snippets/collecting-metrics.md

```diff
@@ -12,14 +12,17 @@ You can also set up the OpenTelemetry Collector to receive metrics from Knative
 !!! warning
     You can't use OpenTelemetry Collector and Prometheus at the same time. The default metrics backend is Prometheus. You will need to remove `metrics.backend-destination` and `metrics.request-metrics-backend-destination` keys from the config-observability Configmap to enable Prometheus metrics.
 
-## About Prometheus
+## About the Prometheus Stack
 
-[Prometheus](https://prometheus.io/) is an open-source tool for collecting,
-aggregating timeseries metrics and alerting. It can also be used to scrape the OpenTelemetry Collector that is demonstrated below when Prometheus is used.
+[Prometheus Stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) is a preconfigured collection of Kubernetes manifests, Grafana dashboards, and Prometheus rules, combined to provide end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator. The stack includes by default some Prometheus packages and Grafana.
```

Comment:
Let's put this after Prometheus and Grafana.

Reply:
The section titles are okay? I'm not sure about _Import Grafana dashboard_

Reply:
Let's move "Import Grafana dashboards" after "Access the Grafana instance locally". These should be at the same level.